Formula/borgbackup-fuse.rb

```diff
@@ -79,6 +79,11 @@ def install
     ENV["BORG_LIBZSTD_PREFIX"] = Formula["zstd"].prefix
     ENV["BORG_OPENSSL_PREFIX"] = Formula["openssl@1.1"].prefix
     virtualenv_install_with_resources
+
+    man1.install Dir["docs/man/*.1"]
+    bash_completion.install "scripts/shell_completions/bash/borg"
+    fish_completion.install "scripts/shell_completions/fish/borg.fish"
+    zsh_completion.install "scripts/shell_completions/zsh/_borg"
```

Comment:
Looks good, thanks for the PR!

Did you test installing borg using this and man pages / shell completions worked afterwards?

Reply:
Yes. The man pages work and I tested the completions on zsh. Followed the instructions brew gives you about completions and it worked. The 4 lines are just copied from the Formula in the main repo, so I assume it works on the other shells